corehq/apps/locations/static/locations/js/location_drilldown.async.js

```diff
@@ -171,6 +187,26 @@ function LocationModel(data, root, depth, func, withAllOption) {
         }
     };
 
+    this.toPlainJS = function() {
```

Comment:
I see, github hid the diff for that file because it was large.  Can you move this function into the ICDS code?  It's not used outside of ICDS, so it's a little confusing for it to be in the product code.